impl/src/main/java/org/ehcache/internal/serialization/CompactJavaSerializer.java

```diff
@@ -150,12 +150,6 @@ protected int getOrAddMapping(ObjectStreamClass desc) throws IOException {
     }
   }
 
-  @Override
-  public void close() {
```

Comment:
I believe this logic should remain here, and be called when `SerializationProvider.releaseSerializer` is called.
This could be achieved by having our default serialisers implementations still implement `Closeable` or some other interface.